src/gateways/NextAuthUserSessionGateway.ts

```diff
@@ -1,8 +1,7 @@
-import { GetServerSidePropsContext } from "next";
 import { getSession } from "next-auth/client";
 import { UserSessionGateway } from "./interfaces/UserSessionGateway";
 
-export interface Session {
+export interface BeaconsSession {
```

Comment:
Getting confused between the `Session` we define here and a `Session` type exported by next-auth.  Chose to rename ours.